comp_loinc/build.py

```diff
@@ -27,40 +47,54 @@ def build_codes(
         schema_file: str,
         code_file: str,
         output: str):
+    """Build codes.
+    :param schema_file: Path to...
+    :param code_file: Path to...
+    :param output: Path where output will be saved."""
     cp = CodeIngest(schema_path=schema_file, code_file_path=code_file)
     cp.generate_codes()
     cp.write_output_to_file(output_path=output)
 
+
 @app.command(name='composed')
 def build_composed_classes(
         schema_file: str,
         composed_classes_data_file: str,
         output: str):
-    """
-    TODO this is just calling a linkml owl cli, should be written as code
-    :param schema_file:
-    :param code_file:
-    :param output:
-    :return:
-    """
+    """Build composed classes.
+    :param schema_file: Path to...
+    :param composed_classes_data_file: Path to...
+    :param output: Path where output will be saved."""
+    # TODO: this is just calling a linkml owl cli, should be written as code
```

Comment:
@Tim Moved your TODO here, as it was previously showing up in `--help` because it was in the docstring.